lib/concurrent-ruby/concurrent/executor/java_executor_service.rb

```diff
@@ -96,6 +96,8 @@ def newThread(runnable)
         return thread
       end
     end
-    
+
+    private_constant :DaemonThreadFactory
```

Comment:
can `private_constant` be accessed by subclasses?

Reply:
yeah that works fine. (It has to be private otherwise it would show up in the documentation.)